content/docs/command-reference/commit.md

```diff
@@ -36,23 +36,26 @@ further detailed below.
 - It's always possible to manually execute the source code used in a stage
   without DVC (outputs should be unprotected or removed first in certain cases,
   see `dvc unprotect`). Once a desirable result is reached, use `dvc add` or
-  `dvc commit` as appropriate to update DVC-files and store changed data to the
-  cache.
+  `dvc commit` as appropriate to update the corresponding stage (or `.dvc` file)
+  and store changed data to the cache.
 
 - Sometimes we want to edit source code, config, or data files in a way that
   doesn't cause changes in the results of their data pipeline. We might write
   add code comments, change indentation, remove some debugging printouts, or any
   other change that doesn't cause changed stage outputs. However, DVC will
   notice that some <abbr>dependencies</abbr> and have changed, and expect you to
   reproduce the whole pipeline. If you're sure no pipeline results would change,
-  just use `dvc commit` to force update the related DVC-files and cache.
+  just use `dvc commit` to force update the related stage (or `.dvc` file) and
+  cache.
 
 Let's take a look at what is happening in the first scenario closely. Normally
 DVC commands like `dvc add`, `dvc repro` or `dvc run` commit the data to the
-<abbr>cache</abbr> after creating a DVC-file. What _commit_ means is that DVC:
+<abbr>cache</abbr> after creating or updating the `dvc.yaml` (or `.dvc` file).
+What _commit_ means is that DVC:
```

Comment:
OK, this is closer but still not quite right. Is dvc.yaml the relevant special file to mention?

Reply:
Actually `dvc.lock` and `.dvc` will be updated if we run `repro` after doing changes in some files(deps).
And `dvc.yaml` will be updated in case we run a stage for first time and use ` --no-commit` also.

Reply:
Here I think instead of mentioning all three(which I did currently), it can be `updating the stage or .dvc file`:sweat_smile:

Reply:
"updating the stage" would be incorrect. Again, `commit` does not change the stage at all. A stage = a command, its dependencies (paths), and outputs (paths).

Reply:
Continued in https://github.com/iterative/dvc.org/pull/1626#pullrequestreview-460938171.